integration-tests/src/main/java/org/apache/druid/testing/utils/QueryResultVerifier.java

```diff
@@ -19,12 +19,17 @@
 
 package org.apache.druid.testing.utils;
 
+import org.apache.druid.java.util.common.StringUtils;
+import org.apache.druid.java.util.common.logger.Logger;
+
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 
 public class QueryResultVerifier
 {
+  private static final Logger LOG = new Logger(QueryResultVerifier.class);
+
   public static boolean compareResults(
```

Comment:
I think this method should throw an exception when there is a mismatch else every caller would have to do that handling.